js/jquery-focussable.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
+import $ from 'jquery';
+
+if ($.ui === undefined) {
+    $.ui = {};
+}
+
+// Support: IE 8 only
+// IE 8 doesn't resolve inherit to visible/hidden for computed values
+function visible( element ) {
+    var visibility = element.css( "visibility" );
+    while ( visibility === "inherit" ) {
+        element = element.parent();
+        visibility = element.css( "visibility" );
+    }
+    return visibility !== "hidden";
+}
+
+$.ui.focusable = function( element, hasTabindex ) {
```

Comment:
I don't think there's much benefit to having this available generally or to pollute jQuery with the jQuery-UI namespace, it's only likely to cause pain. Can we just make this a private function that can be called from the selector extension below?

Reply:
Yeh I'l do that

Reply:
Done it